tests/tests_insights_autoupdate.yml

```
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Basic insights autoupdate test
+  hosts: all
+  gather_facts: true
+  become: true
+
+  tasks:
+    - name: Skip when running with embedded Candlepin
+      meta: end_play
+      when:
+        - lookup('env', 'LSR_RHC_TEST_DATA') | length == 0
+      delegate_to: 127.0.0.1
+      become: false
+
+    - name: Import test data
+      include_vars:
+        file: "{{ lookup('env', 'LSR_RHC_TEST_DATA') }}"
+      delegate_to: 127.0.0.1
+      become: false
+
+    - name: Test autoupdate
+      block:
+        - name: Add autoupdate and register insights
+          include_role:
+            name: linux-system-roles.rhc
```

```suggestion
            name: linux-system-roles.rhc
            public: true
```
Otherwise, I don't think the `__rhc_insights_conf` variable will be exported from the role

---

you don't need to add `public: true` below - once it is exported, it is available for the rest of the playbook